Source/JavaScriptCore/dfg/DFGClobberize.h

```diff
@@ -797,6 +791,7 @@ void clobberize(Graph& graph, Node* node, const ReadFunctor& read, const WriteFu
     case ResolveScopeForHoistingFuncDeclInEval:
     case ResolveScope:
     case ToObject:
+    case ToThis:
```

Comment:
Why does `ToThis` become much more conservative here? (clobber top)

Reply:
Reverted, and adjusted AI.